c10/core/DispatchKeySet.cpp

```diff
@@ -4,6 +4,19 @@
 
 namespace c10 {
 
+// On x86 architecture atomic operations are lock-less.
```

Comment:
We also build on ARM. But a relaxed concurrency should be acceptable here. More below.